utils/music_crawlers.py

```
@@ -0,0 +1,934 @@
+    # URL级别去重
+    seen_urls = set()
+    unique_results = []
+    for item in all_results:
+        if item['url'] not in seen_urls:
+            unique_results.append(item)
+            seen_urls.add(item['url'])
+    
+    # 使用缓存进行短期去重
+    unique_results = music_cache.filter_duplicates(unique_results)
+    
+    # 去重后可能为空，需要修正返回语义
```

Ruff (0.15.4)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)